doc/source/conf.py

```diff
@@ -302,11 +307,15 @@
         "json_url": f"https://{cname}/versions.json",
         "version_match": switcher_version,
     },
-    "use_meilisearch": {
-        "api_key": os.getenv("MEILISEARCH_PUBLIC_API_KEY", ""),
-        "index_uids": {
-            f"pymapdl-v{switcher_version.replace('.', '-')}": "PyMAPDL",
-        },
+    # "use_meilisearch": {
+    #     "api_key": os.getenv("MEILISEARCH_PUBLIC_API_KEY", ""),
+    #     "index_uids": {
+    #         f"pymapdl-v{switcher_version.replace('.', '-')}": "PyMAPDL",
+    #     },
+    # },
+    "cheatsheet": {
+        "file": "cheat_sheet/cheat_sheet.qmd",
+        "title": "PyMAPDL cheat sheet",
```

Comment:
So the cheatsheet is generated with the docs. Is it uploaded as part of the docs artifacts??

Reply:
Yes, you can find it within the `documentation-html` folder under `_static/cheat_sheet.pdf` as it's built on-the-fly for the documentation,